WcaOnRails/db/migrate/20160627215744_create_competition_events.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+class CreateCompetitionEvents < ActiveRecord::Migration
+  def up
+    create_table :competition_events do |t|
+      t.string :competition_id, limit: 32, null: false
+      t.string :event_id, limit: 6, null: false
```

Comment:
As I think was already discussed elsewhere, The Rails Way seems to be to do application level validations rather than database level validations. As a DB guy, I totally get that you don't trust us to not do something evil through phpMyAdmin, but I would ask that for consistency's sake, we remove this `limit`s here.

(Someday later if lack of database validations proves to bite us in the ass, I'd be open to discuss adding them to all our strings in the db.)